morpheus/models/dfencoder/autoencoder.py

```diff
@@ -787,6 +810,396 @@ def fit(self, df, epochs=1, val=None, run_validation=False, use_val_for_loss_sta
             i_loss = cce_loss[:, i]
             self.feature_loss_stats[ft] = self._create_stat_dict(i_loss)
 
+    def fit_distributed(
+        self, 
+        train_dataloader, 
+        rank, 
+        world_size, 
```

Comment:
not sure how to set `rank` or `world_size`. can they have default values or more doc to explain how to set them?

Reply:
rank and world_size are variables provided to use when we set up multi processes. Sharing a script I used to run this offline.